### PR TITLE
Add latte-integrale

### PR DESCRIPTION
**Pull request by saraedum**
see https://github.com/conda-forge/sage-feedstock/issues/42

### CODE REVIEW

**Comment by saraedum** [comment]
@mkoeppe @videlec, please ping me here if you want to be a maintainer of this as well.

**Comment by saraedum** [comment]
`make check` fails for me. `test-library-examples` also fails for me locally.

**Comment by saraedum** [comment]
@mkoeppe: I am seeing a
```
division by zero in _ntl_gdiv
FAIL test-library-examples (exit status: 134)
```

Am I doing something wrong in my `make check`?
[test-suite.log](https://github.com/conda-forge/staged-recipes/files/3441501/test-suite.log)

**Comment by saraedum** [comment]
@slel @videlec If you know something about LattE, feel free to weigh in here :)

**Comment by mkoeppe** [comment]
> @mkoeppe: I am seeing a
> 
> ```
> division by zero in _ntl_gdiv
> FAIL test-library-examples (exit status: 134)
> ```
> 
> Am I doing something wrong in my `make check`?
> [test-suite.log](https://github.com/conda-forge/staged-recipes/files/3441501/test-suite.log)

I haven't seen this error before. The test-suite including test-library-examples run successfully on latte's Travis CI, see for example https://travis-ci.com/latte-int/latte/jobs/220281717

File: recipes/latte-integrale/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    - automake
+    - libtool
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - perl
+  host:
+    - ntl
+    - gmp
+    - cddlib
+    # The 4ti2 interface (which only calls the 4ti2 binaries) is only built if
```
Comment (mkoeppe) [inline recipes/latte-integrale/meta.yaml]: Actually LattE uses the 4ti2 library, not the binaries, for example in `code/latte/triangulation/RegularTriangulationWith4ti2.cpp`

Comment (saraedum) [reply recipes/latte-integrale/meta.yaml]: Thanks, fixed in https://github.com/conda-forge/latte-integrale-feedstock/pull/1.

**Pull request merged**